src/python/psic/resizer/resize.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import os
+from typing import Union
+
+import pytest
+
+from psic import s
+from psic.resizer.generate import ResizeImages
+
+
+@pytest.mark.skip
+class ResizeRunner:
```

Comment:
Declaring skip as class is redundant as the file is already ignored in the codecov configuration yaml, should be removed really